capabilities.json

```diff
@@ -1,538 +1,434 @@
-                        "displayName": "Arial",
-                        "value": "Arial"
-                    },
-                    {
-                        "displayName": "Arial Black",
-                        "value": "Arial Black"
-                    },
-                    {
-                        "displayName": "Arial Unicode MS",
-                        "value": "Arial Unicode MS"
-                    },
-                    {
-                        "displayName": "Calibri",
-                        "value": "Calibri"
-                    },
-                    {
-                        "displayName": "Cambria",
-                        "value": "Cambria"
-                    },
-                    {
-                        "displayName": "Cambria Math",
-                        "value": "Cambria Math"
-                    },
-                    {
-                        "displayName": "Candara",
-                        "value": "Candara"
-                    },
-                    {
-                        "displayName": "Comic Sans MS",
-                        "value": "Comic Sans MS"
-                    },
-                    {
-                        "displayName": "Consolas",
-                        "value": "Consolas"
-                    },
-                    {
-                        "displayName": "Constantia",
-                        "value": "Constantia"
-                    },
-                    {
-                        "displayName": "Corbel",
-                        "value": "Corbel"
-                    },
-                    {
-                        "displayName": "Courier New",
-                        "value": "Courier New"
-                    },
-                    {
-                        "displayName": "Georgia",
-                        "value": "Georgia"
-                    },
-                    {
-                        "displayName": "Lucida Sans Unicode",
-                        "value": "Lucida Sans Unicode"
-                    },
-                    {
-                        "displayName": "Symbol",
-                        "value": "Symbol"
-                    },
-                    {
-                        "displayName": "Tahoma",
-                        "value": "Tahoma"
-                    },
-                    {
-                        "displayName": "Times New Roman",
-                        "value": "Times New Roman"
-                    },
-                    {
-                        "displayName": "Trebuchet MS",
-                        "value": "Trebuchet MS"
-                    },
-                    {
-                        "displayName": "Verdana",
-                        "value": "Verdana"
-                    },
-                    {
-                        "displayName": "Wingdings",
-                        "value": "Wingdings"
+    ],
+    "objects": {
+        "general": {
+            "displayName": "General",
```

Comment:
delete all displayName and displayNameKey from capabilites.json. It should be handled in the formatting model

Reply:
Done 2b680b2